slack_backup/download.py

```diff
@@ -85,8 +85,11 @@ def download(self, url, filetype):
                 logging.debug("File `%s' already exist, skipping", filepath)
                 os.unlink(filepath)
         else:
+            if temp_file == ".gif":
```

Comment:
I never bump on file with only an exception. If this is a use case for you, than we probably don't want to hardcode an extension to the check. Rather, I would expect, that we do the check, like:

```python

import os
...

basename, ext = os.path.splitext(temp_file)
if not basename:
    temp_file = 'generic' + ext
```

Furthermore, `generic` name might be a subject for config option.

Reply:
that makes sense